src/main/java/org/openmuc/jmbus/DataRecord.java

```diff
@@ -250,10 +297,154 @@ else if (vif == 0xfd) {
+                    break;
+                case 0b010_1111:
+                    descriptionExtensions.add(DescriptionExtension.PER_KELVIN);
+                    break;
+                case 0b011_0000:
+                    descriptionExtensions.add(DescriptionExtension.PER_KILO_WATT_HOUR);
+                    break;
+                case 0b011_0001:
+                    descriptionExtensions.add(DescriptionExtension.PER_GIGA_JOULE);
+                    break;
+                case 0b011_0010:
+                    descriptionExtensions.add(DescriptionExtension.PER_KILO_WATT);
+                    break;
+                case 0b011_0011:
+                    descriptionExtensions.add(DescriptionExtension.PER_KELVIN_LITER);
+                    break;
+                case 0b011_0100:
+                    descriptionExtensions.add(DescriptionExtension.PER_VOLT);
+                    break;
+                case 0b011_0101:
+                    descriptionExtensions.add(DescriptionExtension.PER_AMPERE);
+                    break;
+                case 0b011_0110:
+                    descriptionExtensions.add(DescriptionExtension.MULTIPLIED_BY_SECOND);
+                    break;
+                case 0b011_0111:
+                    descriptionExtensions.add(DescriptionExtension.MULTIPLIED_BY_SECOND_PER_VOLT);
+                    break;
+                case 0b011_1000:
+                    descriptionExtensions.add(DescriptionExtension.MULTIPLIED_BY_SECOND_PER_AMPERE);
+                    break;
+                case 0b011_1001:
+                    descriptionExtensions.add(DescriptionExtension.START_DATE);
+                    break;
+                case 0b011_1010:
+                    descriptionExtensions.add(DescriptionExtension.UNCORRECTED_UNIT);
+                    break;
+                case 0b011_1011:
+                    descriptionExtensions.add(DescriptionExtension.POSITIVE_ACCUMULATION);
+                    break;
+                case 0b011_1100:
+                    descriptionExtensions.add(DescriptionExtension.NEGATIVE_ACCUMULATION);
+                    break;
+                case 0b011_1110:
+                    descriptionExtensions.add(DescriptionExtension.BASE_CONDITION);
+                    break;
+                case 0b100_0000:
+                    descriptionExtensions.add(DescriptionExtension.LOWER_LIMIT);
+                    break;
+                case 0b100_1000:
+                    descriptionExtensions.add(DescriptionExtension.UPPER_LIMIT);
+                    break;
+                case 0b111_0000:
+                case 0b111_0001:
+                case 0b111_0010:
+                case 0b111_0011:
+                case 0b111_0100:
+                case 0b111_0101:
+                case 0b111_0110:
+                case 0b111_0111:
+                    multiplierExponent += -6 + (buffer[i] & 0x7);
+                    break;
+                case 0b111_1101:
```

Comment:
Lt. Doku https://m-bus.com/documentation-wired/08-appendix gibt es hier noch den Fall, den wir berücksichtigen könnten:

E111 10nn | Additive correction constant: 10nn-3 · unit of VIF (offset)